src/main/kotlin/fi/metatavu/timebank/api/controllers/SynchronizeController.kt

```diff
@@ -40,17 +46,22 @@ class SynchronizeController {
      * Synchronizes time entries from Forecast to Time-bank
      *
      * @param after YYYY-MM-DD LocalDate
-     * @return Length of entries synchronized
      */
-    suspend fun synchronize(after: LocalDate?): List<TimeEntry> {
-        return try {
-            val forecastPersons = personsController.getPersonsFromForecast()
+    suspend fun synchronize(after: LocalDate?) {
+        try {
+            var forecastPersons = personsController.getPersonsFromForecast()
 
             val worktimeCalendars = forecastPersons.map { person ->
                 worktimeCalendarController.checkWorktimeCalendar(person)
             }
 
-            val entries = retrieveAllEntries(after, worktimeCalendars)
+            forecastPersons = personsController.filterPersons(forecastPersons)
```

Comment:
Is there any good reasons for exposing the filter method from personsController instead of exposing filtered resources from the controller and leaving the filter method as private?

Reply:
It is done that way to be able to create WorktimeCalendars for each person to mitigate the possibility of having TimeEntries persisted without WorktimeCalendar and then later persons are filtered to prevent creating of empty TimeEntries for persons no longer active (or system users or clients or sub contractors). 
Possible solutions that comes to my mind:
- Not persist entries of persons who no longer work/are active
- Create a "default" WorktimeCalendar for TimeEntries to use if for some reason there isn't a WorktimeCalendar associated with it
- Do not filter the persons at all and handle prevention of entry creation another way.